src/main/clojure/clara/rules/compiler.clj

```diff
@@ -31,7 +31,9 @@
             Accumulator
             ISystemFact]
```

Comment:
I'll have to think about if there are any ways this batching could interact negatively (i.e. in a bug-causing way) with node sharing.  We might need to add some tests around that.

Reply:
I don't see any problem here on review.